src/components/app-rating-mobile/AppRatingMobile.tsx

```diff
@@ -11,16 +11,17 @@ import { TotalReviews, UserRoleEnum } from '~/types'
 
 interface AppRatingMobileProps extends RatingProps {
   reviewsCount: TotalReviews
+  hideStar?: boolean
 }
 
-const AppRatingMobile: FC<AppRatingMobileProps> = ({ value, reviewsCount }) => {
+const AppRatingMobile: FC<AppRatingMobileProps> = ({ value, reviewsCount, hideStar = false }) => {
```

Comment:
and better idea will be to use comments separately , because of this whole component you are using only comments

Reply:
Here i didn't understand

Reply:
we have component whish has star , rating and rewiews and use of it only rewievs part with wrapper of box, when we need only typography with amount of reviews

Reply:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/84423165/9edfad58-9b4a-4b50-a354-bc3cdafed0c6)
U mean to do like that?